css-brochure-form/styles.css

```diff
@@ -73,11 +72,8 @@ button {
 }
 
 .button {
-  display: flex;
-  justify-items: right;
-  justify-content: right;
-  align-content: flex-end;
-  position: absolute;
-  top: 325px;
-  left: 670px;
+  justify-content: flex-end;
+  position: relative;
+  top: 280px;
+  right: 50px;
```

Comment:
What happens when you don't apply these? They shouldn't be necessary 🤔

Reply:
The button stays in the goes to the top right ;)

Reply:
![Screen Shot 2021-06-25 at 1 08 18 PM](https://user-images.githubusercontent.com/52221756/123479908-70503180-d5b6-11eb-9053-78123aeb9c65.png)

Reply:
This is what happens when I remove the offsets.